configure

```diff
@@ -280,7 +282,7 @@ fi
 # cross-compiling to one of these OSes then you'll need to specify
 # the correct CPU with the --cpu option.
 case $targetos in
-AIX|OpenBSD)
+AIX|*BSD)
```

Comment:
@bvanassche Won't this also disable it on FreeBSD?

Reply:
Is FreeBSD already supported? From the configure script:

    $ grep targetos= configure 
    targetos=""
      targetos="Android"
      targetos="Linux"
      targetos='OpenBSD'
      targetos='NetBSD'
      targetos='SunOS'
      targetos='CYGWIN'
      targetos=`uname -s`

Reply:
FreeBSD is supported and works, we just don't check for it specifically. As such, the change would not have disabled lex on FreeBSD, but I still think the targetos check should just target netbsd and openbsd specifically.

Reply:
@bvanassche it is - it just falls into the `uname -s` case.

Reply:
Oh yes, good point, it is actually broken as-is.

Reply:
@axboe Thanks for fixing that one up (I'm not a FreeBSD user, just a curious bystander)!

Reply:
Thanks for the review, good catch!